docs/UserGuide.md

```diff
@@ -1,186 +1,175 @@
 
 0. Ensure you have Java version `1.8.0_60` or later installed in your Computer.<br>
-
    > Having any Java 8 version is not enough. <br>
-   > This app will not work with earlier versions of Java 8.
+   This app will not work with earlier versions of Java 8.
 
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds.
+1. Download the latest `todolist.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your ToDo List.
+3. Double-click the file to start the app. The GUI should appear in a few seconds. 
    > <img src="images/Ui.png" width="600">
 
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
+   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will display help. 
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
+   * **`add`** : adds a task (e.g. `add buy some milk  d/tomorrow p/high`)
+   * **`edit`** : edits an existing task (e.g. `edit 1 buy some milk and cheese p/low`)
+   * **`kill`** : deletes a task in the current list (e.g. `kill 1`)
+   * **`quit`** : quits the app (e.g. `quit`)
 6. Refer to the [Features](#features) section below for details of each command.<br>
 
 
-## 2. Features
+## Features
 
 > **Command Format**
->
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
 
-### 2.1. Viewing help : `help`
-
+#### Viewing help : `help / h`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+ 
+#### Adding a task: `add / a`
+Adds a task to the todo list<br>
+Format: `add TASK [d/DEADLINE] [p/PRIORITY] [t/TAGNAME]` 
 
-### 2.2. Adding a person: `add`
-
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
-
-> Persons can have any number of tags (including 0)
-
-Examples:
-
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
-
-### 2.3. Listing all persons : `list`
-
-Shows a list of all persons in the address book.<br>
-Format: `list`
+> Tasks can have any number of tags (including 0)
 
-### 2.4. Editing a person : `edit`
+Examples: 
+* `add Buy donuts d/tomorrow p/high t/work`
+* `add Buy milk d/Friday p/low t/home`
 
```

Comment:
- How about covering the basic date format for example instead of natural language kind of date (today, tomorrow, wednesday) first?
- How about the other types of tasks mentioned in the project scope? Do include examples of them for the users.
- Do you have an option for time?